### PR TITLE
Optimize logging docs

### PR DESCRIPTION
**Pull request by wangyikewxgm**
### Description of your changes

1. change the logging example to avoid printing too many logs.
2. introduce to visualization docs to tell user how to import a grafana dashboard 

I have:

- [ ] Read and followed KubeVela's [contribution process](https://github.com/kubevela/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] Update `sidebar.js` if adding a new page.
- [ ] Run `yarn start` to ensure the changes has taken effect.

### CODE REVIEW

**Review by Somefive — APPROVED** [review]

File: docs/platform-engineers/operations/o11y/logging.md [hunk 1]
```diff
@@ -144,7 +154,7 @@ After the application is created, you can find the deployment resource created b
 
 ### nginx access log analysis
 
-If your application is an nginx gateway, the `stdout-logs` trait provide the capability to parse nginx [combined](https://docs.nginx.com/nginx/admin-guide/monitoring/logging/) format log to json format as follows:
+If your application is a nginx gateway, the `stdout-logs` trait provide the capability to parse nginx [combined](https://docs.nginx.com/nginx/admin-guide/monitoring/logging/) format log to json format as follows:
```
Comment (wonderflow) [inline docs/platform-engineers/operations/o11y/logging.md]:
```suggestion
If your application is an nginx gateway, the `stdout-logs` trait provide the capability to parse nginx [combined](https://docs.nginx.com/nginx/admin-guide/monitoring/logging/) format log to json format as follows:
```

**Review by wonderflow — APPROVED** [review]

File: i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/operations/o11y/logging.md [hunk 2]
```diff
@@ -213,7 +223,7 @@ spec:
 ```
 
 该例子中，除了将 nginx 输出的 `combinded` 日志转换成 json 格式，并为每条日志增加一个 `new_field` 的 json key ，json value 的值为 `new value`。具体 vector VRL 如何编写请参考[文档](https://vector.dev/docs/reference/vrl/)。
-如果你针对这种处理方式，专门了特殊的日志分析大盘，可以参考 [文档](../observability) 将其导入到 grafana 中。
+如果你针对这种处理方式，制作了专门的日志分析大盘，可以参考 [文档](./visualization#dashboard-customization) 提供的三种方式将其导入到 grafana 中。
```
Comment (wonderflow) [inline i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/operations/o11y/logging.md:226]: 这个链接在中文模式下不work

Comment (wangyikewxgm) [reply i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/operations/o11y/logging.md:226]: 现在 visualization 的中文文档还只有英文的，这个链接测试是 work 的。

Comment (wonderflow) [reply i18n/zh/docusaurus-plugin-content-docs/current/platform-engineers/operations/o11y/logging.md:226]: OK

**Pull request merged**